tests/unit/types/test_documentset.py

```diff
@@ -1,11 +1,13 @@
 from jina import Document
 from jina.types.sets import DocumentSet
 from tests import random_docs
+from tests import random_docs_new_api
```

Comment:
please merge master branch to your branch, `random_docs_new_api ` has been merged back to `random_docs` in #1356

Reply:
Just remark: I prefer rebasing, since it makes a cleaner commit history. Anyhow, since we mostly "squash and merge" the difference is minor.